docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -99,7 +99,11 @@ The alert summary section provides general alert details such as the alert's sta
 
 The alert's reason statement is under the alert summary. The reason statement describes what generated the alert and provides general alert details. You can use this information to understand the alert's origin and determine if the alert is relevant to your investigation.
 
-The *Highlighted Fields* section is also provided for all alerts and is customized to only show the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
+Also described are the number of cases to which the alert has been added. An alert can be added to zero or more cases. Click case names to open case details.
+
+The *Highlighted Fields* section is also provided for all alerts and displays the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
+
+Additional data about the alert's highlighted fields is given in the *Alert prevalence* column. This column shows the total number of other alerts within the selected timeframe that have identical values. For example, imagine an alert with an alert prevalence value of 3 for the `host.name` field. This means, within the given timeframe, three other alerts with the same `host.name` value exist. Alert prevalence data can help you investigate relationships with other alerts and gain more context about the event producing the alert. 
```

Comment:
This passage contains a lot of passive voice and discursive phrasing that could be streamlined.

* For the sake of conciseness, I don't think it's necessary to say how many cases an alert can be attached to, esp since in this context the user isn't trying to attach anything; they're just looking at info. Also adding something to "zero or more" is kind of impossible, so it could be omitted.
* The definition of what the prevalence column shows is already clear and straightforward, and the example seems kind of repetitive and wordy.

```suggestion
The alert details flyout also lists the number and names of cases to which the alert has been added. Click case names to open case details.

The *Highlighted Fields* section displays the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.

The *Alert prevalence* column shows the total number of other alerts within the selected timeframe that have identical values. Alert prevalence data can help you investigate relationships with other alerts and gain more context about the event producing the alert. 
```

Reply:
Thanks for these edits, @joepeeples ! I debated adding the sentence about adding an alert because it didn't seem fully relevant to what was being described. It's also a bit misleading because the option to do that is in a sep menu. 

Thanks as well for your thoughts on the paragraph about the **Alert prevalence** column. I can see how the first sentence in the **Alert prevalence** section is unnecessary given the paragraph's proximity to the **Highlighted Fields** description--good call on removing that. I'm still torn on removing the example. I actually think it adds some context to the following sentence, which describes how to use alert prelavence data.

Reply:
Thanks for the reminder, @joepeeples ! Verified that the value in the prevalence column refers to the total number of alerts in the timeframe (so all other alerts and the current one). Will make that change in the next round of commits.